plugins/lime-plugin-fbw/src/containers/scanPage/ScanList.js

```diff
@@ -20,22 +19,48 @@ export const ScanList = ({
 }) => {
 
 	const [expandedAps, setExpandedAps] = useState([]) 	// Expand scanned lists
+	const [getStatus, setGetStatus] = useState(false) 	// Activate get status
```

Comment:
It was difficult to me to understand the purpose of this `getStatus` state.
Is it to enabling/disabling backend status polling?
If so, here there is an example from other component that does the same: https://github.com/libremesh/lime-app/blob/a62547d971438cf3197f40432708469b410eca94/plugins/lime-plugin-firmware/src/upgradePage/upgradePage.js#L33

I think a better naming as in this example could help.

Reply:
5e54a4f it has more sense your proposed way, the code is cleaner